docs/latex/non_admin.tex

```diff
@@ -0,0 +1,35 @@
+\documentclass{article}
+
+% Language setting
+% Replace `english' with e.g. `spanish' to change the document language
+\usepackage[english]{babel}
+
+% Set page size and margins
+% Replace `letterpaper' with`a4paper' for UK/EU standard size
+\usepackage[letterpaper,top=2cm,bottom=2cm,left=3cm,right=3cm,marginparwidth=1.75cm]{geometry}
+
+% Useful packages
+\usepackage{amsmath}
+\usepackage{graphicx}
+\usepackage[colorlinks=true, allcolors=blue]{hyperref}
+
+\title{GUFI: A Common User's Guide}
+\author{Braeden Slade}
+
+\begin{document}
+\maketitle
+
+\input{sections/introduction.tex}
```

Comment:
switch to `\include` and you won't need `\clearpage`

Reply:
So for some of the formatting, i needed to use input, but I did go back and use include where appropriate

Reply:
What causes this? `\begin{tabular}{l|p{6cm}}`?